### ISSUE TITLE
When you map Start to R2 it also triggers ZR

### ISSUE THREAD

**Issue by dfilskov** [weihuoya/citra#662]
On XP Plus using latest Citra 20220907 I when I map the thin controller (the one with buttons only: •, 1, 2, 3, 4 on the front) 's right trigger to START it keeps activating both Start and RZ when pressed.

Even when RZ is mapped to another button - e.g. the button named 4.

Either there's a hardcode mapping to the trigger or earlier mappings aren't cleared properly, I guess.

**Comment by Tappy-tech** [comment]
In config-mmj.ini file, how are buttons mapped?

**Comment by dfilskov** [comment]
Took me quite a few attempts to reproduce the issue. I had to map several buttons and move them around afterwards.

**Comment by dfilskov** [comment]
Here's how it looks in the Settings UI

![Screenshot_20220914-230446~2](https://user-images.githubusercontent.com/15096058/190263890-4d178760-7765-43be-b9b3-6c26949954d4.png)
![Screenshot_20220914-230456~2](https://user-images.githubusercontent.com/15096058/190263896-d63602f5-e270-4e46-a343-1b93946c3fcc.png)
![Screenshot_20220914-230456~3](https://user-images.githubusercontent.com/15096058/190263897-ac4d414f-a215-4c32-9f76-d8a82a967c49.png)
![Screenshot_20220914-230503~2](https://user-images.githubusercontent.com/15096058/190263898-931b19a1-f0d7-4ed0-a3c8-bca0bc27f4ee.png)
![Screenshot_20220914-230509~3](https://user-images.githubusercontent.com/15096058/190263901-f4bfcd34-9c8a-4c84-8ce3-9c5bef51b9bc.png)

**Comment by dfilskov** [comment]
Here's a video of what happens - notice the game both pauses (the small screen switches to showing Items instead of a map) and triggers a whirling weapon.

https://www.dropbox.com/s/bxzhah2z8iventg/item-view_and_whirl-weapon.mkv?dl=0

**Comment by Tappy-tech** [comment]
> [Controls] button_a = code:108 button_b = code:96 button_down = code:16,dir:+ button_home = code:106 button_l = code:102 button_left = code:15,dir:- button_r = code:103 button_right = code:15,dir:+ button_start = code:100 button_up = code:16,dir:- button_x = code:97 button_y = code:109 button_zl = code:104 circle_pad_x = code:0,dir:+ circle_pad_y = code:1,dir:- [Renderer] landscape_layout_option = 2

Looks same as in Settings Ui... I fear that it's controller issues, although, I didn't look up much.

**Comment by dfilskov** [comment]
Can I somehow test if it's a controller issue with the XP Plus?

**Comment by dfilskov** [comment]
Wait ... the ZR isn't mapped at all. So how can it be an issue with the controller and not the software?

**Comment by Tappy-tech** [comment]
> Wait ... the ZR isn't mapped at all. So how can it be an issue with the controller and not the software?

I think because of how it is interpreted by Citra MMJ, if you don't have issues with other emulators.

**Comment by dfilskov** [comment]
Yeah. Wrong interpretation I'd say since I'm quite sure I don't have this issue in other emulators.

Hey - if I map ZR to another button than R2 and Iclear Y the other button triggers both Y and ZR. Really odd.

Like this:
![Screenshot_20220914-234005~2](https://user-images.githubusercontent.com/15096058/190268313-7ecdc397-c44a-4538-975f-adbd4b98c2b6.png)

... but notice that the other button which is not R2 is also called 109 - like R2.
